server/__init__.py

```diff
@@ -58,5 +58,6 @@
 
 if CONSTANTS['SOCIAL_PLATFORM'] == 'discord':
     social = DiscordDriver(
-        CONSTANTS['SOCIAL_API_URL']
+        CONSTANTS['SOCIAL_API_URL'],
+        CONSTANTS['TESTIMONIAL_API_URL']
```

Comment:
```suggestion
        CONSTANTS['TESTIMONIAL_API_URL'],
```
This is extremely petty and 1000% optional, but just thought I'd share what is purely my opinion: I like adding commas in these cases so if you ever have to add another argument, it's only one line change in git, as opposed to one change and one addition.  Just thought I'd throw this out there as a thing in case you like it